### PR TITLE
Fix Widgets/Workers documentation

### PR DESCRIPTION
**Pull request by bampakoa**
- Make `viewerDragDropMixin` a `constructor`
- Tag `cesiumWorkerBootstrapper` methods as `function`

### CODE REVIEW

File: Source/Widgets/Viewer/viewerDragDropMixin.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ define([
      * Rather than being called directly, this function is normally passed as
      * a parameter to {@link Viewer#extend}, as shown in the example below.
      * @exports viewerDragDropMixin
+     * @constructor
```
Comment (hpinkos) [inline Source/Widgets/Viewer/viewerDragDropMixin.js]: `viewerDragDropMixin` isn't a constructor

File: Source/Workers/cesiumWorkerBootstrapper.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ var requirejs, require, define;
     }
 
     /**
+     * @function
```
Comment (hpinkos) [inline Source/Workers/cesiumWorkerBootstrapper.js]: Instead of making these changes. mark this file `@private`

**Comment by bampakoa** [comment]
@hpinkos this is ready

**Comment by hpinkos** [comment]
👍 Thanks @bampakoa!

**Pull request merged**